apps/editor.planx.uk/src/pages/FlowEditor/components/NewSettings/TeamSettings/Contact/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { useSlackMessage } from "pages/FlowEditor/components/Settings/hooks/useSlackMessage";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { ChangeEvent } from "react";
+import InputLabel from "ui/editor/InputLabel";
+import Input from "ui/shared/Input/Input";
+
+import SettingsFormContainer from "../../shared/SettingsForm";
+import { GET_TEAM_SETTINGS, UPDATE_TEAM_SETTINGS } from "./queries";
+import { validationSchema } from "./schema";
+import {
+  ContactFormValues,
+  GetTeamSettingsData,
+  UpdateTeamSettingsVariables,
+} from "./types";
+
+export const TeamContactSettings: React.FC = () => {
+  const [teamId, teamSlug] = useStore((state) => [
+    state.teamId,
+    state.teamSlug,
+  ]);
+
+  const { mutate: sendSlackMessage } = useSlackMessage();
+
+  return (
+    <SettingsFormContainer<
+      GetTeamSettingsData,
+      UpdateTeamSettingsVariables,
+      ContactFormValues
+    >
+      query={GET_TEAM_SETTINGS}
+      queryVariables={{ slug: teamSlug }}
+      mutation={UPDATE_TEAM_SETTINGS}
+      getInitialValues={({ teams: [team] }) => team.settings}
+      getMutationVariables={(values) => ({
+        teamId,
+        settings: {
+          help_email: values.helpEmail,
+          help_phone: values.helpPhone,
+          help_opening_hours: values.helpOpeningHours,
+          homepage: values.homepage,
+        },
+      })}
+      validationSchema={validationSchema}
+      legend="Contact information"
+      description={
+        <>
+          Populates Gov UK Notify templates for email replies and footer contact
+          information. Gov UK Notify templates apply to all submission services.
+        </>
+      }
+      onSuccess={(values, data) => {
+        const oldEmail = data?.teams[0].settings.helpEmail;
+        const hasEmailUpdated = oldEmail && values.helpEmail !== oldEmail;
+        if (hasEmailUpdated) {
+          const message = `:e-mail: *${teamSlug}* updated their Gov UK Notify reply-to email to *${values.helpEmail}*; ensure this is configured in Gov UK Notify dashboard.`;
+          sendSlackMessage(message);
+        }
+      }}
+    >
+      {({ formik }) => (
+        <>
+          <InputLabel label="Homepage URL" htmlFor="homepage">
+            <Input
+              name="homepage"
+              onChange={(e: ChangeEvent<HTMLInputElement>) =>
+                formik.setFieldValue("homepage", e.target.value)
+              }
+              value={formik.values.homepage}
+              errorMessage={formik.errors.homepage}
+              id="homepage"
+            />
+          </InputLabel>
```

Comment:
In future we could look at a "builder" pattern here, where you pass an array of fields to the component and they're rendered in a standard way - 

```ts
export const ContactForm = createSettingsForm({
  query: GET_TEAM_SETTINGS,
  mutation: UPDATE_TEAM_SETTINGS,
  legend: "Contact information",
  fields: [
    { name: "helpEmail", type: "email", label: "Reply-to email" },
    { name: "helpPhone", type: "tel", label: "Phone number" },
    // ...
  ],
});
```

I've not gone for this here as - 
 - it's a bigger refactor, and will slow down the completion of this task
 - it's not really needed (?) - we don't have that many forms, and there's a decent level of novelty between fields
 - it's easily reachable as a next step if we find the number of pages starts to grow